scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ResumingClientHandshaker.java

```diff
@@ -281,7 +282,7 @@ public void startHandshake() throws HandshakeException {
 		state = message.getMessageType().getCode();
 		clientHello = message;
 
-		DTLSFlight flight = new DTLSFlight(getSession());
+		DTLSFlight flight = new DTLSFlight(getSession(), 1);
```

Comment:
I think, this will be the first CLIENT_HELLO (flight 1). 
The second CLIENT_HELLO will be send in ClientHandshaker.receivedHelloVerifyRequest (flight 3). So the numbers should fit.